components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsLoginAttemptsMap/src/IsAnalyticsLoginAttemptsMap.jsx

```diff
@@ -20,29 +20,14 @@
 import React from 'react';
 import Widget from '@wso2-dashboards/widget';
 import VizG from 'react-vizgrammar';
-import { MuiThemeProvider, createMuiTheme } from '@material-ui/core';
-import Switch from '@material-ui/core/Switch';
-import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { MuiThemeProvider } from 'material-ui';
```

Comment:
Lets import both MuiThemeProvider & Toggle in one statement here

Reply:
Done